packages/types/src/ui/components/popover.ts

```diff
@@ -0,0 +1,6 @@
+export enum PopoverAlign {
+  Left = "left",
+  Right = "right",
+  LeftOutside = "left-outside",
+  RightOutside = "right-outside",
+}
```

Comment:
`PopoverAlignment` already is an enum exported from BBUI - so you can resue that instead of adding this! 👌